### PR TITLE
Add unit tests for google-cloud component

### PR DESCRIPTION
**Pull request by SinaChavoshi**

### CODE REVIEW

File: prow/prowjobs/kubeflow/pipelines/kubeflow-pipelines-presubmits.yaml [hunk 1]
```diff
@@ -171,6 +171,16 @@ presubmits:
         command:
         - ./test/presubmit-components-gcp.sh
 
+  - name: kubeflow-pipelines-components-google-cloud-python38
+    cluster: build-kubeflow
+    decorate: true
+    run_if_changed: "^(components/google-cloud/.*)|(/pipelines/components/google-cloud/tests/presubmit.sh)$"
```
Comment (Bobgy) [inline prow/prowjobs/kubeflow/pipelines/kubeflow-pipelines-presubmits.yaml:177]: Presubmit test script is already in the first folder, so can we simplify?

File: prow/prowjobs/kubeflow/pipelines/kubeflow-pipelines-presubmits.yaml [hunk 2]
```diff
@@ -171,6 +171,16 @@ presubmits:
...
+    spec:
+      containers:
+      - image: python:3.8
+        command:
+        - ./pipelines/components/google-cloud/tests/presubmit.sh
```
Comment (Bobgy) [inline prow/prowjobs/kubeflow/pipelines/kubeflow-pipelines-presubmits.yaml:182]: You can write

./components/google-cloud/tests/presubmit.sh

Current working dir is repo root

File: prow/prowjobs/kubeflow/pipelines/kubeflow-pipelines-presubmits.yaml (hunk 1 above)
Comment (SinaChavoshi) [reply prow/prowjobs/kubeflow/pipelines/kubeflow-pipelines-presubmits.yaml:177]: good point, done :)

**Comment by Bobgy** [comment]
/lgtm
/approve
Thank you @SinaChavoshi for setting up unit tests!

**Pull request merged**